test/gurka/test_gtfs.cc

```diff
@@ -200,16 +200,18 @@ TEST(GtfsExample, WriteGtfs) {
   }
 
   // write agency.txt
-  struct Agency ttc {
-    .agency_id = a1_id, .agency_name = "Toronto Commission", .agency_url = "http://www.ttc.ca",
```

Comment:
This file was using designated initializers - which is C++20. Some C++17 compilers allows it, even though it's not C++17. MSVC however fails hard.